README.md

```diff
@@ -52,12 +88,17 @@ Terragrunt supports the following use cases:
 1. [Work with multiple Terraform modules](#work-with-multiple-terraform-modules)
 
 
+### Keep your Terraform code DRY
 
+* [Motivation](#motivation)
+* [Remote Terraform configurations](#remote-terraform-configurations)
+* [How to use remote configurations](#how-to-use-remote-configurations)
+* [DRY Terraform code and immutable infrastructure](#dry-terraform-code-and-immutable-infrastructure)
```

Comment:
Consider changing this to "**Achieving** DRY Terraform code and immutable infrastructure."